File: takeItEasy-application/src/main/kotlin/com/sudosoo/takeItEasy/application/common/service/CommonService.java
package com.sudosoo.takeItEasy.application.common.service;

import org.springframework.http.HttpStatus;
import org.springframework.web.server.ResponseStatusException;

public class CommonService {
    public static <E> boolean checkNotNullData(E e, String message) {
        if (e == null) {
            throw new ResponseStatusException(HttpStatus.BAD_REQUEST, message);
        }
        return true;
    }
}

File: takeItEasy-application/src/main/kotlin/com/sudosoo/takeItEasy/application/core/commons/CommonService.kt
package com.sudosoo.takeItEasy.application.commons

import com.fasterxml.jackson.databind.ObjectMapper
import org.springframework.http.HttpStatus
import org.springframework.web.server.ResponseStatusException

object CommonService {

    fun getObjectMapper(): ObjectMapper {
        return ObjectMapper()
    }
    fun <E> checkNotNullData(e: E?, message: String?): Boolean {
        if (e == null) {
            throw ResponseStatusException(HttpStatus.BAD_REQUEST, message)
        }
        return true
    }
}
